### PR TITLE
Add web spring

### PR DESCRIPTION
**Pull request by AndrewPolishchuc**

### CODE REVIEW

**Review by andromash — APPROVED** [review]
Nice!

File: src/main/java/movie/storage/service/impl/CinemaHallServiceImpl.java [hunk 1]
```diff
@@ -2,15 +2,19 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
 
 @Service
 public class CinemaHallServiceImpl implements CinemaHallService {
-    @Inject
-    private CinemaHallDao cinemaHallDao;
+    private final CinemaHallDao cinemaHallDao;
+
+    @Autowired
```
Comment (Sofasmile) [inline src/main/java/movie/storage/service/impl/CinemaHallServiceImpl.java:14]: should we use Autowired above constructor or not?
let's create our project in one style

File: src/main/java/movie/storage/service/impl/ShoppingCartServiceImpl.java [hunk 2]
```diff
@@ -4,23 +4,28 @@
...
+import org.springframework.stereotype.Service;
 
 @Service
 public class ShoppingCartServiceImpl implements ShoppingCartService {
-    @Inject
     private ShoppingCartDao shoppingCartDao;
-    @Inject
     private TicketDao ticketDao;
-    @Inject
     private UserDao userDao;
```
Comment (Sofasmile) [inline src/main/java/movie/storage/service/impl/ShoppingCartServiceImpl.java]: do we need userDao here?

**Review by Sofasmile — COMMENTED** [review]
let's remove HibernateUtil

File: src/main/java/movie/storage/service/impl/ShoppingCartServiceImpl.java (hunk 2 above)
Comment (AndrewPolishchuc) [reply src/main/java/movie/storage/service/impl/ShoppingCartServiceImpl.java]: Deleted

File: src/main/java/movie/storage/service/impl/CinemaHallServiceImpl.java (hunk 1 above)
Comment (AndrewPolishchuc) [reply src/main/java/movie/storage/service/impl/CinemaHallServiceImpl.java:14]: I used above all service constructors in the same style.

**Review by Sofasmile — APPROVED** [review]

File: src/main/java/movie/storage/service/impl/ShoppingCartServiceImpl.java (hunk 2 above)
Comment (Sofasmile) [reply src/main/java/movie/storage/service/impl/ShoppingCartServiceImpl.java]: and what about dao?

**Pull request merged**